src/app/forms/templates/test-records/section-templates/test/specialist-test-section-group1.template.ts

```diff
@@ -46,8 +46,14 @@ export const SpecialistTestSectionGroup1: FormNode = {
             {
               name: 'testResult',
               label: 'Result',
-              editType: FormNodeEditTypes.HIDDEN,
               viewType: FormNodeViewTypes.HIDDEN,
+              editType: FormNodeEditTypes.RADIO,
+              options: [
```

Comment:
The options used for `testResult` are the same throughout. Would you not consider putting them in a util folder within `forms > templates` and just reference that in each instance?